pkg/application/inject/fuse/mount_point_script.go

```diff
@@ -0,0 +1,151 @@
+package fuse
+
+import (
+	"context"
+	"github.com/fluid-cloudnative/fluid/pkg/common"
+	"github.com/fluid-cloudnative/fluid/pkg/ddc/base"
+	"github.com/fluid-cloudnative/fluid/pkg/scripts/poststart"
+	"github.com/fluid-cloudnative/fluid/pkg/utils"
+	"github.com/fluid-cloudnative/fluid/pkg/utils/kubeclient"
+	corev1 "k8s.io/api/core/v1"
+	"strings"
+)
+
+func (s *Injector) injectCheckMountReadyScript(pod common.FluidObject, runtimeInfos map[string]base.RuntimeInfoInterface) error {
+	objMeta, err := pod.GetMetaObject()
+	if err != nil {
+		return err
+	}
+
+	if !utils.FuseSidecarUnprivileged(objMeta.Labels) {
```

Comment:
I think it does nothing with privileged mode.

Reply:
Done. Added injection for init containers